deepchecks/tabular/utils/task_inference.py

```diff
@@ -0,0 +1,101 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Utils module containing functionalities to infer the task type and possible label classes."""
+
+__all__ = ['get_possible_classes', 'infer_task_type']
+
+from typing import List, Optional
+
+import pandas as pd
+from sklearn.base import ClassifierMixin
+
+from deepchecks import tabular
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.tabular.utils.feature_inference import is_categorical
+from deepchecks.tabular.utils.task_type import TaskType
+from deepchecks.utils.strings import is_string_column
+from deepchecks.utils.typing import BasicModel
+
+
+def get_possible_classes(model: Optional[BasicModel], train_dataset: 'tabular.Dataset',
+                         test_dataset: Optional['tabular.Dataset'] = None, force_multiclass: bool = False) \
```

Comment:
When is force_multiclass used? Why not name it force_classification?

Reply:
we could name it force classification, will do